### PR TITLE
Add coverage on project

### PR DESCRIPTION
**Pull request by ViniciusAlmeidaSilva**
coverage

solves #17

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 require 'omnitrade_client'
 require 'webmock/rspec'
+require 'simplecov'
+SimpleCov.start
```
Comment (adbatista) [inline spec/spec_helper.rb]: simplecov need to be required before the lib code

Comment (adbatista) [reply spec/spec_helper.rb]: https://github.com/colszowka/simplecov#getting-started

> Load and launch SimpleCov at the very top of your test/test_helper.rb (or spec_helper.rb, rails_helper, cucumber env.rb, or whatever your preferred test framework uses):

**Review by adbatista — CHANGES_REQUESTED** [review]

**Review by sobrinho — APPROVED** [review]

**Comment by adbatista** [comment]
fix the conflicts pls

**Review by davidsilveira — APPROVED** [review]

**Review by davidsilveira — APPROVED** [review]
Looks good ;)

**Comment by davidsilveira** [comment]
@ViniciusAlmeidaSilva I have an observation, you could exclude the `'/spec'` folder from this verification. I believe it's unnecessary to verify whether this folder is covered or not.
@sobrinho and @adbatista what do you think about it?

**Comment by adbatista** [comment]
sure, remove it

**Review by adbatista — APPROVED** [review]

**Comment by andrerpbts** [comment]
👏

**Pull request merged**